code/modules/power/fusion/fusion_reactions.dm

```diff
@@ -160,8 +160,8 @@ proc/get_fusion_reaction(var/p_react, var/s_react, var/m_energy)
 /decl/fusion_reaction/boron_hydrogen
 	p_react = "boron"
 	s_react = GAS_HYDROGEN
-	minimum_energy_level = FUSION_HEAT_CAP * 0.5
+	minimum_energy_level = 15000
```

Comment:
Can you expand on why you're removing 75,000,000, half of the core of the sun's temperature, in favor of 15,000?

Reply:
Because it's literally not feasible. the RUST literally never surpasses 300 thousand Kelvin. That's how we had it pre-Bay.